libs/common/setup.go

```diff
@@ -58,6 +60,16 @@ func Setup(serviceName, version string, auth bool) {
 
 		setupAuth()
 	}
+
+	organizationIdStr := hwutil.GetEnvOr("ORGANIZATION_ID", "")
+	if organizationIdStr != "" {
+		organizationID, err := uuid.Parse(organizationIdStr)
+		if err != nil {
+			log.Fatal().Err(err).Msg("invalid uuid for environment variable ORGANIZATION_ID")
+		}
+		log.Info().Str("organizationID", organizationID.String()).Msg("using fallback organizationID")
```

Comment:
```suggestion
		log.Info().Str("organizationID", organizationID.String()).Msg("using default organizationID")
```
it's not really a fallback, we expect the header to not be set in a deployment that uses it

Reply:
When the `X-Organization` header is not set, we are trying to _fallback_ to this environment variable. I've added a comment anyway around this code.